src/components/elements/MapPolygonPanel/MapEditPolygonPanel.tsx

```diff
@@ -0,0 +1,95 @@
+import { t } from "@transifex/native";
+import classNames from "classnames";
+import { Dispatch, SetStateAction } from "react";
+import { When } from "react-if";
+
+import Icon, { IconNames } from "@/components/extensive/Icon/Icon";
+
+import Button from "../Button/Button";
+import Text from "../Text/Text";
+import AttributeInformation from "./AttributeInformation";
+import ChecklistInformation from "./ChecklistInformation";
+import VersionInformation from "./VersionInformation";
+
+export interface MapEditPolygonPanelProps {
+  setEditPolygon: Dispatch<SetStateAction<boolean>>;
+  tabEditPolygon: string;
+  setTabEditPolygon: Dispatch<SetStateAction<string>>;
+  setPreviewVersion: Dispatch<SetStateAction<boolean>>;
+}
+
+const MapEditPolygonPanel = ({
+  setEditPolygon,
+  tabEditPolygon,
+  setTabEditPolygon,
+  setPreviewVersion
+}: MapEditPolygonPanelProps) => {
+  return (
+    <>
+      <div className="flex items-start justify-between gap-4">
+        <div>
+          <Text variant="text-12-light" className="text-white ">
+            Faja Lobi Project
+          </Text>
+          <Text variant="text-20-bold" className="mb-4 text-white">
+            Iseme Site
```

Comment:
Please remove all mocked data before merging to staging

Reply:
done